blog-mysql-seed/mysql-seeder/seed.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO User (UserID, FirstName) VALUES (1, "Peter");
```

Comment:
Not a SQL pro, but capital letters in column names looks weird to me at first glance. I'm more used to snake case for SQL in general, not sure if that's the standard or not. IIRC SQL is not case sensitive, so snake case is maybe better here?